librz/include/rz_util/rz_str.h

```diff
@@ -147,8 +150,12 @@ RZ_API int rz_str_cmp(const char *dst, const char *orig, int len);
 RZ_API int rz_str_casecmp(const char *dst, const char *orig);
 RZ_API int rz_str_ncasecmp(const char *dst, const char *orig, size_t n);
 RZ_API int rz_str_ccpy(char *dst, char *orig, int ch);
-RZ_API const char *rz_str_get(const char *str);
-RZ_API const char *rz_str_get2(const char *str);
+static inline const char *rz_str_get(const char *str) {
+	return str ? str : rz_str_null_c;
+}
+static inline const char *rz_str_get2(const char *str) {
```

Comment:
I don't like this name too much. It doesn't really tell me the diff with the other without 2

Reply:
`rz_str_get_safe()` ?

Reply:
Maybe I'd call this one rz_str_get_null and the other rz_str_get

Reply:
Sgtm

Reply:
Did renaming in ee77324 5174f9d a794239 611065b.